crypto/evp_extra/p_hkdf.c

```diff
@@ -207,6 +207,8 @@ const EVP_PKEY_METHOD hkdf_pkey_meth = {
     pkey_hkdf_derive,
     /*paramgen=*/NULL,
     pkey_hkdf_ctrl,
+    /*encapsulate*/NULL,
```

Comment:
This order of comment then null or null then comment is silly. I see that you're just being consistent